ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java

```diff
@@ -5406,16 +5406,16 @@ protected Table getTableObjectByName(String tabName, boolean throwException) thr
     String[] names = Utilities.getDbTableName(tabName);
     final String  tableName = names[1];
     final String  dbName = names[0];
-    String metaTable = null;
+    String tableIdentifier = null;
```

Comment:
The below logic looks similar to `TableName.fromString`, could we reuse/optimize here?

Reply:
Done. I reused  `TableName.fromString`  to simplify here.

Reply:
I finally gave up reusing `TableName.fromString` here as i found some failed tests.

There are some code snippets use `catalog.dbName.tblName` to identify a table and `TableName.fromString` can not distinguish between 
 `catalog.dbName.tblName` and iceberg's  `dbName.tblName.metaTable`.  I have no better idea to handle this now.

Reply:
that's fine